haystack/pipelines/standard_pipelines.py

```diff
@@ -714,27 +714,43 @@ def __init__(self, document_store: BaseDocumentStore):
         self.pipeline.add_node(component=document_store, name="DocumentStore", inputs=["Query"])
         self.document_store = document_store
 
-    def run(self, document_ids: List[str], top_k: int = 5):
+    def run(
+        self,
+        document_ids: List[str],
+        filters: Optional[Dict[str, Union[Dict, List, str, int, float, bool]]] = None,
+        top_k: int = 5,
+        index: Optional[str] = None,
+    ):
         """
         :param document_ids: document ids
+        :param filters: Optional filters to narrow down the search space to documents whose metadata fulfill certain conditions
         :param top_k: How many documents id to return against single document
+        :param index: Optional name of index where the documents shall be written to. If None, the DocumentStore's default index (self.index) will be used
```

Comment:
Thanks for the documentation! However, could we update this to 
`
Optionally specify the name of index to query the document from. If None, the DocumentStore's default index (self.index) will be used.
`

Reply:
Done